src/user/user.c

```diff
@@ -35,8 +42,30 @@ int main(int ac, char **argv)
                 return 0;
         }
 
-        sleep(20);
+        // get the fd for the map created in the BPF program
+        map_fd = bpf_map__fd(skel->maps.output_map);
+
+        // count the total number of processes executed on the CPUs each second
+        while (1) {
+
```

Comment:
Empty lines should convey meaning by separating logical blocks, in this case it does not have any value so I would just delete that line. (I'm yet to see an empty line at the beginning of a block that has meaning)